airflow/providers/amazon/aws/triggers/emr.py

```diff
@@ -317,3 +317,75 @@ async def run(self) -> AsyncIterator[TriggerEvent]:
                     await asyncio.sleep(int(self.poll_interval))
 
             yield TriggerEvent({"status": "success", "job_id": self.job_id})
+
+
+class EmrStepSensorTrigger(BaseTrigger):
+    """
+    Poll for the status of EMR container until reaches terminal state.
+
+    :param virtual_cluster_id: Reference Emr cluster id
+    :param job_id:  job_id to check the state
+    :param aws_conn_id: Reference to AWS connection id
+    :param poke_interval: polling period in seconds to check for the status
```

Comment:
These parameters are inconsistent with the `__init__`. Or are they for somewhere else?

Reply:
it is indeed incorrect. thank you for pointing this out.